src/drive/web/modules/drive/Fab.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+
+import UiFab from 'cozy-ui/transpiled/react/Fab'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import PlusIcon from 'cozy-ui/transpiled/react/Icons/Plus'
+
+import { AddMenuContext } from 'drive/web/modules/drive/AddMenu/AddMenuProvider'
+
+const useStyles = makeStyles(() => ({
+  root: {
+    position: 'fixed',
+    right: '1rem',
+    bottom: 'calc(var(--sidebarHeight) + 1rem)'
+  }
+}))
```

Comment:
Wondering if it should not be in UI. Since in mobile we always have the sideBar, it should be handled by default?

Reply:
Je me suis posé la même question, je me suis dit qu'on éprouverait d'abord dans l'app, et si jamais à terme oui le rapatrier dans UI si on est sûr de n'avoir que ce comportement